fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/CreateTableCommand.java

```diff
@@ -199,6 +206,20 @@ void handleFallbackFailedCtas(ConnectContext ctx) {
         }
     }
 
+    private String getAutoRangePartitionNameOrNull() {
```

Comment:
maybe return a List<String> add process manual range partition too?

Reply:
this restriction(should be not nullable) is only for auto range partition column. normal range partition column could be nullable when `allow_partition_column_nullable  = true`. so maybe it's not necessary?